src/Microsoft.Deployment.DotNet.Releases/Directory.Build.props

```diff
@@ -0,0 +1,290 @@
+      </PropertyGroup>
+    </When>
+    <When Condition="$(OutputRid.StartsWith('freebsd'))">
+      <PropertyGroup>
+        <TargetsFreeBSD>true</TargetsFreeBSD>
+        <TargetsUnix>true</TargetsUnix>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <PropertyGroup>
+        <TargetsLinux>true</TargetsLinux>
+        <TargetsUnix>true</TargetsUnix>
+      </PropertyGroup>
+      <PropertyGroup Condition="'$(PortableBuild)' == 'true'">
+        <HostMachineRidTargetsDebianPackages Condition="
+          $(HostMachineRid.StartsWith('debian')) or
+          $(HostMachineRid.StartsWith('ubuntu')) or
+          $(HostMachineRid.StartsWith('linuxmint'))">true</HostMachineRidTargetsDebianPackages>
+        <!-- If the build machine isn't known to be Debian-based, try to build RPM packages. -->
+        <HostMachineRidTargetsRpmPackages Condition="'$(HostMachineRidTargetsDebianPackages)' != 'true'">true</HostMachineRidTargetsRpmPackages>
+      </PropertyGroup>
+    </Otherwise>
+  </Choose>
+
+  <PropertyGroup>
+    <CompressedFileExtension Condition="'$(TargetOS)' == 'Windows_NT'">.zip</CompressedFileExtension>
+    <CompressedFileExtension Condition="'$(TargetOS)' != 'Windows_NT'">.tar.gz</CompressedFileExtension>
+  </PropertyGroup>
+
+  <!-- Use actual publishable (non-dummy) package name produced by the build system for this RID -->
+  <PropertyGroup Condition="'$(OutputRid)' != ''">
+    <PackageTargetRid>$(OutputRid)</PackageTargetRid>
+    <PackageTargetRid Condition="'$(OutputRid)' == 'osx.10.11-x64'">osx.10.10-x64</PackageTargetRid>
+    <PackageTargetRid Condition="$(OutputRid.StartsWith('rhel.7.')) and $(OutputRid.EndsWith('-x64'))">rhel.7-x64</PackageTargetRid>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(UsingNETSdkCompiler)' != 'true'">
+    <!-- Disable some standard properties for building our projects -->
+    <NoStdLib>true</NoStdLib>
+    <NoExplicitReferenceToStdLib>true</NoExplicitReferenceToStdLib>
+    <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
+    <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+
+    <!-- We do not want to target a portable profile.
+         TODO: Make this the default in buildtools so this is not necessary. -->
+    <TargetFrameworkProfile></TargetFrameworkProfile>
+    <!-- We set this property to avoid MSBuild errors regarding not setting TargetFrameworkProfile (see above line) -->
+    <PortableNuGetMode>true</PortableNuGetMode>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Never use the NuGet fallback folder that comes with the SDK we use to build.
+         The NuGet fallback folder can/will contain packages we are building in this repo, and we
+         want to ensure we use the correct packages.
+    -->
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <LibPrefix Condition="'$(TargetOS)' != 'Windows_NT'">lib</LibPrefix>
```

Comment:
Is all the stuff in this props file necessary? I don't see any usages of these properties from a quick look.

Reply:
I don't think it's being used right now. @NikolaMilosavljevic when looking at the build pipeline, it seems only WindowsNT debug/release and x86/x64 legs are running, so it should be fine to trim Directory.build.props?

Reply:
We don't build any Linux specific native binaries today and don't have a Linux build leg. Feel free to remove this stuff from your props file as you are not building any native projects.

Reply:
I've cleaned up the props file